exporter/exporter.go

```diff
@@ -21,12 +25,12 @@ const prometheusNamespace = "ebpf_exporter"
 // Exporter is a ebpf_exporter instance implementing prometheus.Collector
 type Exporter struct {
 	config              config.Config
-	modules             map[string]*bcc.Module
+	modules             map[string]*bpf.Module
 	perfMapCollectors   []*PerfMapSink
 	kaddrs              map[string]uint64
 	enabledProgramsDesc *prometheus.Desc
 	programInfoDesc     *prometheus.Desc
-	programTags         map[string]map[string]uint64
```

Comment:
Why remove tags?

Reply:
libbpfgo doesn't have interface to get the tags for program. 
so I changed this to expose the functions

Reply:
Could you open an issue for libbpfgo to add support for that? IIRC, you can only see program tags in `perf` output, so having tags to match those back to ebpf_exporter programs is useful to have.

Reply:
IC, I didn't realized that tag can be used for this.  Let me make libbpfgo  to support tag first.